### PR TITLE
Removed invalid super access tests and created properly syntax error

### LINKED ISSUE #2071: Private access from `super` is invalid syntax
Test `src/class-elements/private-field-super-access-throws.case` is verifying private fields super access. However this is invalid syntax and should throw SyntaxError instead.

Looking private fields proposal, we have changes on [`MemberExpression`](https://tc39.github.io/proposal-class-fields/#prod-MemberExpression), however this rule doesn't include `super.PrivateIdentifier` as valid. Check [https://tc39.github.io/ecma262/#prod-MemberExpression](https://tc39.github.io/ecma262/#prod-MemberExpression) for clarification.

### PR DESCRIPTION
**Pull request by caiolima**
This closes #2071.

### CODE REVIEW

File: src/class-elements/grammar-private-field-super-access.case [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+template: syntax/invalid
+features: [class-fields-private]
+---*/
+
+//- elements
+#m = function() { return 'test262'; };
+
+Child = class extends C {
+  access() {
+    return super.#m;
```
Comment (leobalter) [inline src/class-elements/grammar-private-field-super-access.case]: seems correct to me, but I'd like to confirm with @littledan before landing these tests.

Comment (littledan) [reply src/class-elements/grammar-private-field-super-access.case]: That's right, super.#x is not in the grammar.

Comment (caiolima) [reply src/class-elements/grammar-private-field-super-access.case]: I think we are good to move forward, right?

**Comment by rwaldron** [comment]
@caiolima can you add a second commit with the regenerated tests? Thanks!!

**Comment by caiolima** [comment]
Thx a lot!

**Pull request merged**